### ISSUE TITLE
favorites@cinnamon.org: Filenames in the star menu are percent-encoded

### ISSUE THREAD

**Issue by Anmibe** [linuxmint/cinnamon-spices-applets#4071]
The percent-encoding may decoded before displaying the starred filenames in the menu.

- Linux Mint 20.2

**Comment by mtwebster** [comment]
Can you provide some example filenames? It has no issues displaying non-ascii characters.

**Comment by Anmibe** [comment]
Yes, and even more. As I discovered now, it needs a special condition to reproduce:

- Create a file with e.g. a ß ö ä ü or a space in the name (just one letter is enough): "Test file name öäüß.txt"
- Mark it as a favorite
- Everything is fine!
- Copy (not move) the file or create a new one in a **different folder** with **exactly the same file name**, the path is different
- Mark it as favorite as well 
- Both files will be shown with %-encoding (screenshot)

Control:

- Delete or rename (just delete one letter) one of these files
- %-encoding is decoded

Resume:
%-encoding is missing, when there are favorites with identical filenames in different folders.


![Bildschirmfoto vom 2021-11-29 20-53-44](https://user-images.githubusercontent.com/56979681/143935128-12ba0a3d-9145-495f-bffe-85364b6471a4.png)

**Comment by mtwebster** [comment]
This is a core cinnamon applet, moving to cinnamon..
